memor/prompt.py

```diff
@@ -265,9 +264,10 @@ def load(self, file_path: str) -> None:
         with open(file_path, "r") as file:
             self.from_json(file.read())
 
-    def from_json(self, json_object: Union[str, Dict[str, Any]]) -> None:
+    @staticmethod
+    def validate_extract_json(json_object: Union[str, Dict[str, Any]]) -> None:
```

Comment:
I recommend changing this method to an internal method: `_validate_extract_json`

Reply:
fixed in [abde1b5](https://github.com/openscilab/memor/pull/120/commits/abde1b5ecf3430ebeb79267fdfbe4a8b468b360b)